UserManagement/accounts/domain/models.py

```diff
@@ -0,0 +1,74 @@
+from django.contrib.auth.models import AbstractBaseUser, BaseUserManager, PermissionsMixin
+from django.db import models
+from django.utils import timezone
+
+
+# Custom User Manager
+class CustomUserManager(BaseUserManager):
+    def create_user(self, email, password=None, identity_id=None, **extra_fields):
+        """
+        Creates and saves a User with the given email, identity_id, and password.
+        """
+        if not email:
```

Comment:
if the email is not unique, Django will raise an IntegrityError. You might want to catch this error and provide a more user-friendly message.

Reply:
removed